test/rules/relay_connection_arguments_spec.js

```diff
@@ -122,7 +206,7 @@ describe('RelayConnectionArgumentsSpec rule', () => {
             },
           ],
           message:
-            'Fields that support forward pagination must include a `last` argument that takes a non-negative integer as per the Relay spec.',
+            'Fields that support backward pagination must include a `last` argument that takes a non-negative integer as per the Relay spec.',
```

Comment:
Oops! 🤦‍♂ Thanks for catching this too. That previous message would be super confusing. 😂